.github/workflows/security.yml

```diff
@@ -0,0 +1,39 @@
+name: Scan container images
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 4 * * 1'  # Every Monday at 9:30 IST
+
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: paralusio/dashboard
+
+jobs:
+
+  build-push:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Build an image from Dockerfile (paralus-init)
+        run: |
+          docker build -f "Dockerfile" -t ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ github.sha }} .
```

Comment:
Can be rewrite as `docker build -f "Dockerfile" -t dashboard:${{ github.sha }} .`